ddtrace/contrib/django/patch.py

```diff
@@ -467,7 +468,7 @@ def traced_get_response(django, pin, func, instance, args, kwargs):
         def blocked_response():
             from django.http import HttpResponse
 
-            block_config = core.get_item(HTTP_REQUEST_BLOCKED)
+            block_config = core.get_item(HTTP_REQUEST_BLOCKED) or {}
             desired_type = block_config.get("type", "auto")
             status = block_config.get("status_code", 403)
```

Comment:
```suggestion
            desired_type = "auto"
            status = 403            
            block_config = core.get_item(HTTP_REQUEST_BLOCKED)
            if block_config:
                desired_type = block_config.get("type", desired_type)
                status = block_config.get("status_code", status)
```

Not sure why I don't like the `or {}` bit here 🤔 

I was going to say, maybe `get_item` needs a `default=` so you can do `default={}`


feel free to ignore this suggestion, it is pedantic based on how I'd write it, but there isn't anything wrong with what you have functionally.

Reply:
I prefer the shorter, less verbose solution, especially because here, object in block_config variable is read only.
Small is beautiful? 😄

Reply:
🤷🏻 idk why `or {}` bothers me, it shouldn't 😆